plugins/hls-cabal-project-plugin/src/Ide/Plugin/CabalProject/Orphans.hs

```diff
@@ -0,0 +1,42 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-# LANGUAGE DeriveAnyClass     #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Ide.Plugin.CabalProject.Orphans where
+
+import           Control.DeepSeq
+import           Distribution.Fields.Field
+import           Distribution.Parsec.Position
+-- import Control.DeepSeq (NFData)
+import qualified Distribution.Solver.Types.ProjectConfigPath as PCPath
+import           GHC.Generics                                (Generic)
+
+import qualified Distribution.Client.ProjectConfig.Types     as PC
+
+-- ----------------------------------------------------------------
+-- Cabal-syntax orphan instances we need sometimes
```

Comment:
Could you possibly consider contributing them upstream as well? `Cabal-syntax` already depends on `deepseq`, so it's probably just an oversight.